ts/Series/HollowCandlestick/HollowCandlestickPoint.ts

```diff
@@ -0,0 +1,128 @@
+/* *
+ *
+ *  (c) 2010-2021 Torstein Honsi
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import SeriesRegistry from '../../Core/Series/SeriesRegistry.js';
+import HollowCandlestickSeries from './HollowCandlestickSeries.js';
+import palette from '../../Core/Color/Palette.js';
+import type ColorType from '../../Core/Color/ColorType';
+
+const {
+    seriesTypes: {
+        candlestick: CandlestickSeries
+    }
+} = SeriesRegistry;
+
+/* *
+ *
+ *  Class
+ *
+ * */
+class HollowCandlestickPoint extends CandlestickSeries.prototype.pointClass {
+
+    /* *
+     *
+     *  Properties
+     *
+     * */
+
+    public candleFill: ColorType = void 0 as any;
+
+    public series: HollowCandlestickSeries = void 0 as any;
+
+    /* *
+     *
+     *  Functions
+     *
+     * */
+
+    /* eslint-disable valid-jsdoc */
+
+    public getClassName(): string {
+        let className = super.getClassName.apply(this);
+        const point = this,
+            previousPoint = point.series.points[point.index as any];
```

Comment:
Two things here:
- shouldn't be `point.index - 1`? Right now it compares with itself
  - we need a test for the future, to make sure the correct points have the correct colors
- what about `point.index as any` - is index always defined? What happens when the index is not defined? 🤔  I see `point.index as any` is also used in a11y module. However, `Series.ts` is checking if `index` is defined. It seems `dataSorting`, `dataGrouping`, `boost` and `dragablePoints` modules set this. Maybe @TorsteinHonsi knows more

Reply:
Simple demo shows `point.index` available: https://jsfiddle.net/BlackLabel/5rfb8d3q/3/

Reply:
After some tests, it looks like the optional question mark could be deleted from the `PointLike` interface without harm.